### ISSUE TITLE
Easier access for people who would like to see the material

### ISSUE THREAD

**Issue by blaa** [trifectatechfoundation/teach-rs#134]
I occasionaly conduct various courses, today found reference to teach.rs and wanted to take a peek and see how it looks.

- I started with github repo, cloned it.
- I know the content is in files but it looked like it should be renderable to something.
- README points to modmod/README
- modmod has example command of building. But only partial? Only intro-track?
- It doesn't build with nightly in non-obvious way, had to change toolchain in rustup.
- It builds!
- Hm. What now.
- Ah, there's mdbook command and crutial path information on like 93 of modmod README after some directory listing.
- Should I use mdbook from Debian packages or from cargo? README points to Cargo. Seems to work with Debian package (which I already had, but I know nothing about mdbook)
- It works!

Suggestions:
- Later I found https://teach-rs.trifectatech.org/full/introduction.html that has the material - by googling. But it's not linked in README? It's not official?
- Maybe add some short example of all the commands required to render and start mdbook  in the main README? Or a script?
- Maybe modmod could display mdbook command to display what you've just built?

Certainly it's doable now, just could be simpler for somebody with 15-30 minutes of time. ;)

**Comment by squell** [comment]
1. I've recently had a similar experience (I needed more than 15-30 minutes to be honest, but then I was trying to be really thorough:-) and while doing wrote a small deploy script around `modmod` that I want to port to this repository. That should address this issue.

**Comment by hdoordt** [comment]
Yup this situation definitely needs to improve. As I discussed with @tdittr yesterday, I envision a 'course generator' web-app, on which you can drag modules into your self-made track, and have modmod generate the package. I'll open an issue to discuss this.
